salt/modules/pip.py

```diff
@@ -895,6 +915,24 @@ def install(pkgs=None,  # pylint: disable=R0912,R0913,R0914
     if trusted_host:
         cmd.extend(['--trusted-host', trusted_host])
 
+    if extra_args:
+        # These are arguments from the latest version of pip that
+        # have not yet been implemented in salt
```

Comment:
`# These are any additional arguments to be passed as parameters to the pip command`

I don't think we need to specify that it's for arguments salt hasn't implemented yet